docs/installation.rst

```diff
@@ -118,21 +140,37 @@ If you installed in editable mode as described above, you can update `ESM-Tools`
 Uninstall ESM-Tools
 -------------------
 
-We are sorry to see you go! To uninstall your current installation make sure you have the most recent version of
+We are sorry to see you go! You can uninstall your current `ESM-Tools` installation in two different ways depending slightly on how you installed it.
+
+Manually
+^^^^^^^^
+
+If you have installed `ESM-Tools` with the ``install.sh`` script or using ``pip`` with ``user`` mode, please follow the following steps to uninstall the software manually.
+
+- Delete the ``esm_*`` executables::
+
+    $ rm -ri ~/.local/bin/esm*
+
+- Remove the installed Python packages::
+
+  $ rm -ri ~/.local/lib/python3.<version>/site-packages/esm*
+
+Note that you may have a different Python version, so the second command might need to be adapted.
+
+Using pip
+^^^^^^^^^
+
+Make sure you have the most recent version of
```

Comment:
Is there an specific reason why you are adding all of this? On the past we needed the `pip freeze | grep esm | xargs pip uninstall -y` because we had many independent esm packages but now uninstalling esm_tools it uninstalls everything: I've just tried `pip uninstall esm_tools` and it worked perfectly. The advantage of this is that is only one command and that is independent on whether you are using a conda environment, local environment or direnv.

Reply:
No, I only restructured the section. The pip freeze | grep esm | xargs pip uninstall -y was already in the documentation. Can we than remove it?

Reply:
Maybe you can keep it but only for versions below 6.0.0. For versions >= 6.0.0 `pip uninstall esm_tools` should do.

Reply:
I added here a line for which version of esm-tools the pip uninstall command is appropriate for.